plugins/woocommerce/tests/e2e-pw/fixtures/install-wc.setup.js

```diff
@@ -148,8 +152,14 @@ setup( 'Install WC using WC Beta Tester', async ( { wcbtApi, wpApi } ) => {
 		wpApi
 	);
 
-	if ( finalActivatedWcVersion === resolvedVersion ) {
-		console.log( 'Installing with WC Beta Tester is finished.' );
+	if (
+		wcVersion === 'nightly'
+			? finalActivatedWcVersion.endsWith( '-dev' )
```

Comment:
I don't think `endsWith( '-dev' )` is enough to validate the new version was successfully installed. If the previous version was also a nightly and the new nightly version install fails this will return true. Although, I'm not sure we can find the difference between 2 nightly versions at this point, so I guess it\s better than nothing?

Reply:
I agree it's not an ideal solution, but I couldn't find a way to determine what is the nightly version that will be installed, and compare that to the currently installed version. So I ended with this check, as it's better than nothing. 

After some thinking, I figured out that I could get the version from [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/woocommerce.php), but 1) it will add even more complexity to the script and 2) it's not ideal case as well, because, _I think_, it can happen that versions [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/woocommerce.php) and in the generated nightly build are not the same. What do you think?

Reply:
I cannot think of a good solution, the nightly will have the same version. Let's merge as it is, we can come back and improve it if we can find a better solution.